src/Visualisation-App.tsx

```diff
@@ -46,17 +92,22 @@ const VisualisationApp: React.FunctionComponent = () => {
                         </div>
                         <div className="molstarProteinToggles">
                             <h4>Load</h4>
-                            <button className="optionButtons">
+                            <button className="optionButtons" disabled>
```

Comment:
Why are these buttons permanently disabled?

Reply:
Features haven't been implemented yet. They will be re-enabled in future commits.